src/main/java/app/bpartners/api/service/UserService.java

```diff
@@ -116,4 +119,22 @@ public void deleteUserByEmail(String email) {
     userRepository.deleteById(user.getId());
     cognitoComponent.deleteUserByUsername(email);
   }
+
+  @Transactional
+  public void registerOnStripeActiveUsersWithNullSubscription() {
+    List<User> users = userRepository.getActiveUsersWithNullSubscription();
+    var totalUser = users.size();
+    int[] userNb = {0};
+    users.forEach(
+        user -> {
+          UserNonSubscribed event =
+              UserNonSubscribed.builder()
+                  .userId(user.getId())
+                  .totalNbUser(totalUser)
+                  .userNb(userNb[0])
+                  .build();
+          eventProducer.accept(List.of(event));
```

Comment:
Here the event must be for example, userRegistrationRequested, than rename correctly the event and its compoonent.